.travis/install.sh

```diff
@@ -14,12 +14,13 @@ shlib_sed() {
 # download, compile, and install if it's not already present via travis
 # cache
 if [ -n "${OPENSSL}" ]; then
-    OPENSSL_DIR="ossl-2/${OPENSSL}"
+    OPENSSL_CONFIG_PATH=$(dirname "${BASH_SOURCE[0]}")
```

Comment:
This is really like current location or something, and it can go at the top of the file.